ERROR: Push rejected, failed to compile Node.js app.

Simply followed the instructions in the README. I did set a name for the app and changed YOURAPPNAME in HEROKU URL box. This where the problems start:

```
> ncp node_modules/Casper content/themes/casper
       
       There were errors during the copy.
       Error: ENOENT, lstat '/tmp/build_775a4825961df60db5a3b868394b3806/cobyism-ghost-on-heroku-2581f67/node_modules/Casper'
       
       npm ERR! Linux 3.13.0-100-generic
       npm ERR! argv "/tmp/build_775a4825961df60db5a3b868394b3806/cobyism-ghost-on-heroku-2581f67/.heroku/node/bin/node" "/tmp/build_775a4825961df60db5a3b868394b3806/cobyism-ghost-on-heroku-2581f67/.heroku/node/bin/npm" "install" "--unsafe-perm" "--userconfig" "/tmp/build_775a4825961df60db5a3b868394b3806/cobyism-ghost-on-heroku-2581f67/.npmrc"
       npm ERR! node v0.10.48
       npm ERR! npm  v2.15.1
       npm ERR! code ELIFECYCLE
       npm ERR! ghost-on-heroku@0.11.2 postinstall: `ncp node_modules/Casper content/themes/casper`
       npm ERR! Exit status 1
       npm ERR!
       npm ERR! Failed at the ghost-on-heroku@0.11.2 postinstall script 'ncp node_modules/Casper content/themes/casper'.
       npm ERR! This is most likely a problem with the ghost-on-heroku package,
       npm ERR! not with npm itself.
       npm ERR! Tell the author that this fails on your system:
       npm ERR!     ncp node_modules/Casper content/themes/casper
       npm ERR! You can get information on how to open an issue for this project with:
       npm ERR!     npm bugs ghost-on-heroku
       npm ERR! Or if that isn't available, you can get their info via:
       npm ERR!
       npm ERR!     npm owner ls ghost-on-heroku
       npm ERR! There is likely additional logging output above.
       
       npm ERR! Please include the following file with any support request:
       npm ERR!     /tmp/build_775a4825961df60db5a3b868394b3806/cobyism-ghost-on-heroku-2581f67/npm-debug.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       If you're stuck, please submit a ticket so we can help:
       https://help.heroku.com/
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed

```

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
   "private": true,
   "version": "0.11.2",
   "dependencies": {
-    "Casper": "TryGhost/Casper#1.3.3",
+    "casper": "TryGhost/Casper#1.3.3",
     "ghost": "0.11.2",
     "ghost-s3-storage-adapter": "3.0.4",
     "ncp": "^2.0.0",
@@ -20,7 +20,7 @@
     "node": "~0.10.0"
   },
   "scripts": {
-    "postinstall": "ncp node_modules/Casper content/themes/casper",
+    "postinstall": "ncp node_modules/casper content/themes/casper",
     "start": "node server.js"
   }
 }

